sources/notion/index.js

```diff
@@ -14,6 +14,7 @@ const databases = [
     id: 'd4a935fbb66c46b584061956b17e9335',
   },
 ]
+const baseURL = 'https://www.notion.so/eqproduct'
```

Comment:
maybe make this an env var configurable (and falls back to this particular one?)

Reply:
changed, like this `const { NOTION_TOKEN, BASE_URL = 'https://www.notion.so/eqproduct' } = process.env` right